app/registries/foreman/setting_manager.rb

```diff
@@ -44,11 +50,13 @@ def initialize(storage, context_name, category_name)
       #               default: true,
       #               description: N_('Use Puppet that goes to 11'),
       #               full_name: N_('Use shiny puppet'),
-      #               encrypt: true)
+      #               encrypt: true
```

Comment:
Missing comma here

Reply:
Indeed, thanks, fixed in the origin PR #8470